### PR TITLE
updating of processed files

### PR DESCRIPTION
**Pull request by valentinamignosa**

### CODE REVIEW

**Review by JonPrag — CHANGES_REQUESTED** [review]
Some tiny bits and pieces; very happy to see Himera stuff appearing (I realise I know the man who has taken over as director (Stefano Zangara) from the Egadi project, so must right to him!)

File: inscriptions/ISic020476.xml [hunk 1]
```diff
@@ -170,7 +180,7 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>Ἀριστάρχο̄
+                    <lb n="1"/><persName><name>Ἀριστάρχο̄</name></persName>
```
[the remark this answers is not in the document]
Comment (valentinamignosa) [reply inscriptions/ISic020476.xml:183]: Odd... I can see it, can't you?

File: inscriptions/ISic020478.xml [hunk 2]
```diff
@@ -170,7 +180,7 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>Λιβύο̄
+                    <lb n="1"/><persName><name>Λιβύο̄</name></persName>
```
[the remark this answers is not in the document]
Comment (valentinamignosa) [reply inscriptions/ISic020478.xml:183]: same... let me know if you can see it by opening the file... I don't know why you don't see it

**Comment by JonPrag** [comment]
If it’s there, don’t worry, it just wasn’t visible in the browser and they usually are.

j

**Pull request merged**